include/oneapi/dpl/async

```diff
@@ -19,11 +19,17 @@
 #include "oneapi/dpl/internal/common_config.h"
 #include "oneapi/dpl/pstl/onedpl_config.h"
 
-#include "oneapi/dpl/internal/async_extension_defs.h"
+#if !_ONEDPL_BACKEND_SYCL
+#    error "To use oneDPL asynchronous algorithms, a SYCL compiler is required"
+#endif // !_ONEDPL_BACKEND_SYCL
+
+#if !_ONEDPL_ASYNC_FORWARD_DECLARED
+#    include "oneapi/dpl/internal/async_extension_defs.h"
+#    define _ONEDPL_ASYNC_FORWARD_DECLARED 1
+#endif // !_ONEDPL_ASYNC_FORWARD_DECLARED
 
 #if _ONEDPL_EXECUTION_POLICIES_DEFINED
-// If <execution> has already been included, pull in implementations
-#    include "oneapi/dpl/internal/async_impl/async_impl.h"
+     // If <execution> has already been included, pull in implementations
```

Comment:
// If <execution> has already been included, pull in implementations

Reply:
@akukanov up to you: my proposal to remove empty space in this line:
- like in other places with this comment;
- to fix clang format error.

If you are agree - please apply this change.
Thanks.